src/main/java/Display.java

```diff
@@ -16,4 +14,9 @@ public void show_grid(Grid grid) {
             grid.getSquare(6) + " | " + grid.getSquare(7) + " | " + grid.getSquare(8);
         consoleWriter.println(squaresOutput);
     }
+
+    public void make_move() {
```

Comment:
In Java it's more common to use camel-case, so this should be `makeMove`

Reply:
I'll change this, thanks!